src/Model/ElementForm.php

```diff
@@ -41,18 +41,33 @@ public function ElementForm()
         $current = Controller::curr();
 
         if ($current && $current->getAction() == 'finished') {
-            return $controller->renderWith('ReceivedFormSubmission');
+            return $controller->renderWith(UserDefinedFormController::class .'_ReceivedFormSubmission');
```

Comment:
Is this logic from userforms? I quite like the idea of swapping the `_` for a `\` and making it a new folder and `ReceivedFormSubmission.ss` file, but if it's mimicking userforms logic it's fine :)